src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -67,6 +67,9 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges, O
   // e.g. 'single' or 'multi'.
   @Input() selectionType: string = undefined;
 
+  // If `true` selected item details will be updated on table refresh
+  @Input() updateSelectionOnRefresh = false;
```

Comment:
I suggest to set it to ``true`` by default and force the developer to explicitly set it to ``false`` if he does not like this behaviour for his table.

Reply:
@votdev I've changed the default value to `true`